src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -63,6 +63,12 @@ const propTypes = {
     /** Whether the comment is a thread parent message/the first message in a thread */
     isThreadParentMessage: PropTypes.bool,
 
+    /** Whether the report action type is 'APPROVED' or 'SUBMITTED'. Used to style system messages from Old Dot */
+    isApprovedOrSubmittedReportAction: PropTypes.bool,
+
+    /** Used to format RTL display names in Old Dot system messages e.g. Arabic */
+    shouldConvertToLTR: PropTypes.bool,
```

Comment:
I'm not sure why we need this? I think we always want to convert to LTR!

Reply:
Hmm strange I'm not getting the same

Reply:
Interesting. 

Are you getting the correct result or the wrong one?

Reply:
Ah sorry for not being clearer, I meant I don't see the same issue if every fragment is converted to LTR. Can you double check it's necessary to add the `shouldConvertToLTR`?

Reply:
It feels a bit messy using `shouldConvertToLTR`, as really we're passing `isFirstFragment`. If any other fragment wanted to use `shouldConvertToLTR` it might be a bit unexpected that it's entirely based on the index :smile: So if it is necessary I suggest either renaming or maybe just passing the fragment index instead?

Reply:
No need to spend super long on it though, if you're still getting the same result with converting to LTR on all fragments maybe it's some specific length of text that messes it up for some reason :smile:

Reply:
> Can you double check it's necessary to add the shouldConvertToLTR?

@jjcoffee  I've just tested on a physical device, and yes, the check is necessary for the text to be properly formatted. 

<details>

<summary>Latest screenshot after double checking</summary>

</details>

> So if it is necessary I suggest either renaming or maybe just passing the fragment index instead?

I'll go ahead and rename the prop to `isFragmentContainingDisplayName` which is a bit more descriptive, if that's okay.

Reply:
Huh interesting! The trouble is `isFragmentContainingDisplayName` would return true for all first fragments, so could result in an unexpected result too. `isFirstFragment` is probably the best prop name as it fully describes what's in there :smile: In a way I think I prefer passing the index and making the check in the fragment itself, but don't feel super strongly so I'll leave it up to you.

Reply:
I understand. 

The reason I went with `isFragmentContainingDisplayName` is it explains what fragment we're looking for.  

I was hesitant on using `isFirstFragment` because it could lead to the question "Why is the first fragment so important?" 

If at all something changes in the future and the first fragment is no longer the one containing the display name, `isFirstFragment` wouldn't make much sense, but `isFragmentContainingDisplayName` felt like it would hold pretty well regardless of which position the fragment containing the display name would be, and we'd just need to check the correct index.